src/app/mail/mail-settings/mail-settings.component.ts

```diff
@@ -138,6 +138,9 @@ export class MailSettingsComponent implements OnInit, AfterViewInit, OnDestroy {
   private _filterTimeZone(name) {
     return this.timezones.filter(option => option.value.toLowerCase().indexOf(name.toLowerCase()) > -1);
   }
+  private toggleRecoveyEmailEdit() {
```

Comment:
why this function is private,
you don't need this function.

Reply:
this function is called from multiple buttons, so I have added,
and removed the private identifier.